src/table/TableBodyWrapper.jsx

```diff
@@ -51,29 +51,33 @@ const TableBodyWrapper = ({ tableData, constraints, selectionsAPI, layout, theme
 
   return (
     <TableBody>
-      {rows.map((row) => (
+      {rows.map((row, rowIndex) => (
         <TableRow
           hover={hoverEffect}
           role="checkbox"
           tabIndex={-1}
           key={row.key}
-          className={hoverEffect && classes.hoverTableRow}
+          className={`sn-table-row ${hoverEffect} && ${classes.hoverTableRow}`}
```

Comment:
i think the string becomes `sn-table-row true && hoverTableRow`, think should be

```suggestion
          className={`sn-table-row ${hoverEffect && classes.hoverTableRow}`}
```

Reply:
you are right. Will fix. Thanks.